### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sfcsumb**
Changes for assignment 5

### CODE REVIEW

**Review by denizozgur — CHANGES_REQUESTED** [review]
If you want, you can talk to Prof. for an extension, and we can keep this PR open until you can get the requirements done.

File: cst438_gradebook/src/test/java/com/cst438/EndToEndTestAssignments.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.cst438;
+dpackage com.cst438;
```
Comment (sfcsumb) [inline cst438_gradebook/src/test/java/com/cst438/EndToEndTestAssignments.java]: Fixed

**Review by sfcsumb — COMMENTED** [review]
Fixed the typo

**Review by denizozgur — APPROVED** [review]

**Review by denizozgur — CHANGES_REQUESTED** [review]
If you want, you can talk to Prof. for an extension, and we can keep this PR open until you can get the requirements done.

**Pull request merged**